doc/memos/rdm0.md

```diff
@@ -132,6 +135,12 @@ suggestions.
 - Internet Engineering Task Force ([IETF](http://ietf.org/))
 - Guide to [Contributing to RIOT](https://github.com/RIOT-OS/RIOT/wiki/Contributing-to-RIOT)
 
+## Revision
+
+- Rev2: fixed 80 character limit
+- Rev1: disambiguation of RDM number assignment text
+- Rev0: initial document
```

Comment:
I think @kaspar030 only referred to changes in master, but as an example I'm okay with this